hw/dv/tools/vcs/vcs.mk

```diff
@@ -90,6 +90,13 @@ ifeq (${VCS_COV},1)
   RUN_OPTS    += -assert nopostproc
 endif
 
+# Enable XPROP
+VCS_XPROP ?= 1
```

Comment:
Add `XPROP` in modes.mk to provide a tool-agnostic switch.

Reply:
Change from VCS_XPROP to XPROP. Xcelium xprop support should be in soon